services/wfm/src/main/java/org/openkilda/wfm/topology/flow/bolts/TransactionBolt.java

```diff
@@ -232,6 +232,12 @@ public String getCtrlStreamId() {
         return STREAM_ID_CTRL;
     }
 
+    @Override
+    public AbstractDumpState dumpStateBySwitchId(String switchId) {
+        // Not implemented
```

Comment:
There's no TODO, so I assume this is the expected behavior. If we don't throw an exception here, can we at least log a warning message to notice calling of "not implemented" method?